CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 ## 1.12.1 (Unreleased)
+
+IMPROVEMENTS:
+* Documentation addition for `dome9_continuous_compliance_notification` resource and data source.
```

Comment:
- there is mess here with the links they are pointing to github provider (see old changes)
- your line is missing link